catalog/solutions/zeroboard/2.20.0/solution.json

```diff
@@ -3,6 +3,6 @@
   "name": "Zeroboard",
   "version": "2.20.0",
   "website": "https://zeroboard.jp",
-  "provider": "Zeroboard",
+  "provider_id": "zeroboard",
```

Comment:
@raimundo-henriques - Should the vale be "Zeroboard" instead of zeroboard?

Reply:
Here it should actually be `"zeroboard"`, as the value of `"provider_id"` should be the `user-id` (that only admits lowercase letters) and not the provider name

Reply:
@raimundo-henriques - Thanks! It's clear now.